### PR TITLE
Add telescope eod j2000 option

### PR DESCRIPTION
**Pull request by pidloop**
Add control to allow telescope to use either astrometric J2000 or apparent at current epoch coordinates.

### CODE REVIEW

File: GUI/xephem/fallbacks.c [hunk 1]
```diff
@@ -219,21 +219,22 @@ String fallbacks[] = {
     "XEphem*INDIConfig*EDBValue.value: Mount.GOTOedb.edb",
     "XEphem*INDIConfig*EnableEDB.set: False",
     "XEphem*INDIConfig*EnableRADec.set: False",
+    "XEphem*INDIConfig*TelEOD.set: False",
     "XEphem*INDIConfig*GotoDecOffset.value: 0",
-    "XEphem*INDIConfig*GotoDecSlope.value: .0174532925",
+    "XEphem*INDIConfig*GotoDecSlope.value: .01745329251994329576",
```
Comment (brandon-rhodes) [inline GUI/xephem/fallbacks.c:224]: The decimal expansion `...1994329576` isn't familiar to me and has gotten me curious: why does that particular decimal expansion get connected to all of these floating point values?

**Comment by brandon-rhodes** [comment]
Thank you for putting this pull request together! Are there any users that need to try out this code first and tell us that it works with their telescope? Or, am I cleared to go ahead and land it?

**Comment by pidloop** [comment]
It is good to go. This work began from a request for a new feature from a
user in South Africa controlling his telescope with XEphem. We worked
together both on the requirements and testing after I sent him these
changed files. I also tested against my telescope simulator and both of us
agreed these changes work well.

Along the way we discovered one tiny source of error was in these two
values used for the conversion of dec in degrees and RA in hours to
radians, so I computed them to insane accuracy and pasted them in. Now that
particular limitation will never appear again :-)

**Comment by pidloop** [comment]
I just created another pull request. It affects the same file, scope.c, but
I found it after making the previous PR. I have been waiting for the first
one to be ingested, I wasn't sure if two pull requests could be pending
that affect the same file.

**Pull request merged**